install.sh

```diff
@@ -5,3 +5,12 @@ pip uninstall mathgenerator -y
 
 # Installing package from the current directory
 pip install -e .
+
+# Updating the package list and installing jq and npm
+sudo apt update && sudo apt install -y jq npm
```

Comment:
```suggestion
apt update && apt install -y npm
```
Since the entire script is aimed at managing packages, if a user needs to elevate the command to sudo, they may as well elevate the whole script to sudo.
JQ is a nice utility, but not strictly necessary to install for pm2, why do you suggest it?

Reply:
Make sense.
But I'am not sure why [here](https://github.com/macrocosm-os/prompting?tab=readme-ov-file#how-to-run) is jq.